tests/bdd/ha-bhyve02/test_NAS_T0962.py

```diff
@@ -124,33 +124,35 @@ def on_the_active_directory_page_input_the_domain_name_ad_domain(driver, ad_doma
     global domain
     domain = ad_domain
     assert wait_on_element(driver, 5, xpaths.activeDirectory.title)
-    assert wait_on_element(driver, 7, '//ix-input[@formcontrolname="domainname"]//input', 'inputable')
-    driver.find_element_by_xpath('//ix-input[@formcontrolname="domainname"]//input').clear()
-    driver.find_element_by_xpath('//ix-input[@formcontrolname="domainname"]//input').send_keys(ad_domain)
+    assert wait_on_element(driver, 7, xpaths.activeDirectory.domain_input, 'inputable')
+    driver.find_element_by_xpath(xpaths.activeDirectory.domain_input).clear()
+    driver.find_element_by_xpath(xpaths.activeDirectory.domain_input).send_keys(ad_domain)
 
 
 @then(parsers.parse('input the Account name "{ad_user}", the Password "{ad_password}"'))
 def input_the_account_name_ad_user_the_password_ap_password(driver, ad_user, ad_password):
     """input the Account name "ad_user", the Password "ad_password"."""
-    driver.find_element_by_xpath('//ix-input[@formcontrolname="bindname"]//input').clear()
-    driver.find_element_by_xpath('//ix-input[@formcontrolname="bindname"]//input').send_keys(ad_user)
-    driver.find_element_by_xpath('//ix-input[@formcontrolname="bindpw"]//input').clear()
-    driver.find_element_by_xpath('//ix-input[@formcontrolname="bindpw"]//input').send_keys(ad_password)
+    driver.find_element_by_xpath(xpaths.activeDirectory.account_input).clear()
+    driver.find_element_by_xpath(xpaths.activeDirectory.account_input).send_keys(ad_user)
+    driver.find_element_by_xpath(xpaths.activeDirectory.password_input).clear()
+    driver.find_element_by_xpath(xpaths.activeDirectory.password_input).send_keys(ad_password)
```

Comment:
I think we should have a consistent nomenclature. Either camelCase or Snake_Case but not both. Up above there is "system_dataset" changing to "systemDataset" but here it is "password_input"